packages/terra-functional-testing/package.json

```diff
@@ -26,5 +28,17 @@
   "bugs": {
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
-  "homepage": "https://github.com/cerner/terra-toolkit#readme"
+  "homepage": "https://github.com/cerner/terra-toolkit#readme",
+  "bin": {
+    "tt-test": "bin/test-runner.js"
+  },
+  "dependencies": {
+    "@wdio/cli": "^6.1.4",
+    "@wdio/local-runner": "^6.1.4",
+    "@wdio/mocha-framework": "^6.1.0",
+    "@wdio/spec-reporter": "^6.0.16",
+    "chromedriver": "^81.0.0",
+    "commander": "^5.1.0",
+    "wdio-chromedriver-service": "^6.0.2"
```

Comment:
Do we plan to expand on this as we expand on the capabilities of this package?

Reply:
Potentially, this will be evaluated as we begin to introduce the selenium services and docker. We may end up removing this entirely depending on how our testing strategy evolves (Locally, CI, docker).